src/test/groovy/io/codearte/jfairy/producer/person/PersonSpec.groovy

```diff
@@ -214,5 +231,118 @@ class PersonSpec extends Specification {
 			person.passportNumber()
 	}
 
+	def "withFirstName should create person with specific first name"() {
+		when:
+			def person = fairy.person(withFirstName("Specificfirstname"))
```

Comment:
Please use type in place of `def` where possible.
Please also apply this to other similar places in this PR.